library/analytics/src/main/java/org/cru/godtools/analytics/model/ExitLinkActionEvent.kt

```diff
@@ -7,5 +7,5 @@ private const val FIREBASE_PARAM_EXIT_LINK = "cru_mobileexitlink"
 
 class ExitLinkActionEvent(link: Uri) : AnalyticsActionEvent(action = ACTION_EXIT_LINK) {
     override fun isForSystem(system: AnalyticsSystem) = system == AnalyticsSystem.FIREBASE
-    override val adobeAttributes = mapOf<String, Any>(FIREBASE_PARAM_EXIT_LINK to link.toString())
+    override val firebaseParams = mapOf<String, Any>(FIREBASE_PARAM_EXIT_LINK to link.toString())
```

Comment:
this should be converted to populate a `Bundle` object

Reply:
Sounds good 🙂